solutions/fully-configurable/variables.tf

```diff
@@ -105,6 +105,44 @@ variable "scc_workload_protection_service_plan" {
   }
 }
 
+variable "scc_workload_protection_trusted_profile_name" {
+  description = "The name for the trusted profile that is created by this solution. Must begin with a letter. If a prefix input variable is specified, the prefix is added to the name in the `<prefix>-<name>` format."
+  type        = string
+  default     = "workload-protection-trusted-profile"
+}
+
+##############################################################
+# App Config
+##############################################################
+
+variable "cspm_enabled" {
+  description = "Enable Cloud Security Posture Management (CSPM) for the Workload Protection instance."
```

Comment:
This should mention more information on the trusted profile that is created when its set to true